tests/step-117.sh

```diff
@@ -0,0 +1,178 @@
+#!/bin/sh
+# Test:
+#   Test local urls of shared hooks to not be cloned and updated
```

Comment:
Erm... This is a huge test...
Care to explain what are the things it tests?
Is it possible this is multiple tests in one file? :thinking:

Reply:
Jeah it needs more explanation, but when it fails -> we give (I think) good explanation what we expect.